org.eclipse.winery.bpmn4tosca.converter.tobpel/src/main/java/org/eclipse/winery/bpmn2bpel/parser/Bpmn4JsonParser.java

```diff
@@ -104,32 +115,34 @@ public ManagementFlow parse(URI jsonFileUrl) throws ParseException {
 			/*
 			 * Now since all node models are created they can be linked with each other in the management flow
 			 */
-			log.debug("Building management flow by relating node models...");
-			Iterator<Map.Entry<String, Set<String>>> nodeWithTargetsMapIter = nodeWithTargetsMap.entrySet().iterator();
-			while (nodeWithTargetsMapIter.hasNext()) {
-				Map.Entry<String, Set<String>> inputParamEntry = (Map.Entry<String, Set<String>>) nodeWithTargetsMapIter
-						.next();
-				String srcNodeId = inputParamEntry.getKey();
-				Node srcNode = managementFlow.getNode(srcNodeId);
-				if (srcNode == null) {
-					throw new Exception("Node with id '" + srcNodeId + "' could not be found in the management flow.");
-				}
-
-				/* Relate the source node with its link targets */
-				Iterator<String> nodeTargetIdsIter = inputParamEntry.getValue().iterator();
-				while (nodeTargetIdsIter.hasNext()) {
-					String targetNodeId = (String) nodeTargetIdsIter.next();
-					Node targetNode = managementFlow.getNode(targetNodeId);
-					if (targetNode == null) {
-						throw new Exception("Node with id '" + targetNodeId + "' could not be found in the management flow.");
-					}
-
-					log.debug("Creating link between node with id '" + srcNodeId + "' and target node with id '"
-							+ targetNodeId + "'");
-					managementFlow.addEdge(srcNode, targetNode);
-				}
-			}
-
+//			log.debug("Building management flow by relating node models...");
```

Comment:
Please just remove it. We should have as less as possible disabled code in the code base.